### PR TITLE
Change "experimental" to "beta" in annotations

### PR DESCRIPTION
**Pull request by thockin**
@bgrant0607 @brendandburns

### CODE REVIEW

**Comment by bgrant0607** [comment]
You're going to break existing annotations without backward compatibility?

**Comment by bgrant0607** [comment]
Nevermind, I see the code looks at both.

File: pkg/kubelet/kubelet.go [hunk 1]
```diff
@@ -2838,7 +2838,7 @@ func validateBandwidthIsReasonable(rsrc *resource.Quantity) error {
 }
 
 func extractBandwidthResources(pod *api.Pod) (ingress, egress *resource.Quantity, err error) {
-	str, found := pod.Annotations["kubernetes.io/ingress-bandwidth"]
+	str, found := pod.Annotations["net.beta.kubernetes.io/ingress-bandwidth"]
```
Comment (bgrant0607) [inline pkg/kubelet/kubelet.go]: So, has anyone used these?

Comment (bgrant0607) [inline pkg/kubelet/kubelet.go]: And, if so, why not do the same compatibility dance?

Comment (derekwaynecarr) [inline pkg/kubelet/kubelet.go]: fwiw, i just started evaluating this last week, i cannot imagine there is wide use as it didn't get much in the way of documentation to my knowledge

Comment (bgrant0607) [inline pkg/kubelet/kubelet.go]: How about net.alpha, then?

Comment (thockin) [inline pkg/kubelet/kubelet.go]: That was my feeling too, so I was lazy.  I am fine with alpha, or can do
the compat dance...

**Comment by thockin** [comment]
pushed as "alpha"

**Comment by bgrant0607** [comment]
I'm sure we'll need to rethink these in the future, but good enough for now. 
LGTM

**Pull request merged**